frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -423,409 +424,387 @@ class BitBox02 extends Component<Props, State> {
+                                <Checkbox
+                                    onChange={this.handleDisclaimerCheck}
+                                    className={style.wizardCheckbox}
+                                    id="agreement3"
+                                    checked={agreement3}
+                                    label={t('bitbox02Wizard.backup.userConfirmation3')} />
+                                <Checkbox
+                                    onChange={this.handleDisclaimerCheck}
+                                    className={style.wizardCheckbox}
+                                    id="agreement4"
+                                    checked={agreement4}
+                                    label={t('bitbox02Wizard.backup.userConfirmation4')} />
+                                <Checkbox
+                                    onChange={this.handleDisclaimerCheck}
+                                    className={style.wizardCheckbox}
+                                    id="agreement5"
+                                    checked={agreement5}
+                                    label={t('bitbox02Wizard.backup.userConfirmation5')} />
+                            </ViewContent>
+                            <ViewButtons>
+                                <Button
+                                    primary
+                                    onClick={this.createBackup}
+                                    disabled={creatingBackup || !readDisclaimers}>
+                                    {t('securityInformation.create.button')}
+                                </Button>
+                            </ViewButtons>
+                        </View>
+                    </form>
+                )}
+
+                { (!unlockOnly && appStatus === 'restoreBackup' && status !== 'initialized' && restoreBackupStatus === 'restore') && (
+                    <View
+                        key="restore"
+                        fullscreen
+                        textCenter
+                        withBottomBar
+                        width="700px">
+                        <ViewHeader title={t('backup.restore.confirmTitle')}>
+                        </ViewHeader>
+                        <ViewContent>
+                            <BackupsV2
+                                deviceID={deviceID}
+                                showRestore={true}
+                                showRadio={true}
+                                backupOnBeforeRestore={this.backupOnBeforeRestore}
+                                backupOnAfterRestore={this.backupOnAfterRestore}>
+                                <Button
+                                    transparent
+                                    onClick={this.uninitializedStep}
+                                    disabled={settingPassword}>
+                                    {t('button.back')}
+                                </Button>
+                            </BackupsV2>
```

Comment:
Bug: after selecting a backup and on wrong password confirmation it doesn't show an error, expected `backup.restore.error.general` error.

Issue: before this PR ` <BackupsV2>` never unmounted becuase Steps component only hide the component with CSS. This is why it could catch the error and display in a "Toast" component next time it was visible again.